content/docs/guides/analytics_amp/analytics-vendors@es.md

```diff
@@ -0,0 +1,249 @@
+---
+$title: "Proveedores Analytics"
+$order: 4
+toc: true
+---
+[TOC]
+
+En este documento se enumeran los proveedores de Analytics que tienen configuraciones integradas para su uso con el componente [`amp-analytics`](/es/docs/reference/components/amp-analytics.html).
+
+Al especificar el nombre de un proveedor de análisis con el atributo `type`, puede configurar rápidamente `amp-analytics` para utilizar el producto respectivo. La configuración adicional (como su ID de usuario) puede seguir siendo necesaria.
+
+Puede consultar la documentación de su proveedor si está vinculada en las secciones siguientes.
+
+{% call callout('Tip', type='success') %}
+Si se siente cómodo explorando el código, puede explorar las configuraciones sin procesar del archivo [vendors.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/vendors.js) -en inglés.
+{% endcall %}
+
+**Ejemplo:**
+
+Aquí hay un fragmento que especifica el tipo (`type`) de un proveedor de analytics llamado `XYZ`:
+
+```html
+<amp-analytics type="XYZ"> ... </amp-analytics>
+```
+
+{% call callout('Read on', type='read') %}
+Learn more about tracking analytics with [`amp-analytics`](/docs/reference/components/amp-analytics.html).
```

Comment:
You can remove the English and put the sentence you have below here.  Por favor. :)